google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -1346,9 +1359,11 @@ PooledSession getReadSession() throws SpannerException {
           readWaiters.add(waiter);
         } else {
           span.addAnnotation("Acquired read write session");
+          numSessionsAcquired++;
```

Comment:
@olavloite I added these counters based on my understanding, let me know if there is a better way to get these (acquired and released sessions) values.